maintainers/maintainer-list.nix

```diff
@@ -2742,6 +2742,15 @@
     githubId = 16895361;
     name = "Deniz Alp Durmaz";
   };
+  DAlperin = {
+    email = "git@dov.dev";
+    github = "DAlperin";
+    githubId = 16063713;
+    name = "Dov Alperin";
+    keys = [{
+      fingerprint = "4EED 5096 B925 86FA 1101  6673 7F2C 07B9 1B52 BB61";
```

Comment:
```suggestion
    keys = [{
      longkeyid = "rsa3072/0x7F2C07B91B52BB61";
      fingerprint = "4EED 5096 B925 86FA 1101  6673 7F2C 07B9 1B52 BB61";
```

Reply:
@bobby285271 here it is. Could you throw this in for me? I don't have a commit bit on nixpkgs so I can't actually modify this pr.